plugin-server/tests/shared/process-event.ts

```diff
@@ -337,7 +346,7 @@ export const createProcessEventTests = (
         if (database === 'clickhouse') {
             expect(queryCounter).toBe(11 + 14 /* event & prop definitions */)
         } else if (database === 'postgresql') {
-            expect(queryCounter).toBe(14 + 14 /* event & prop definitions */)
+            expect(queryCounter).toBe(12 + 14 /* event & prop definitions */)
```

Comment:
What is this?

Reply:
hijacked your branch and deleted this 💩 

I had gotten rid of it in a PR that never got merged. It had a purpose, but nobody remembers what that was anymore - I've asked around.